dlp/api/Snippets.Tests/SendDataToTheHybridJobTriggerTests.cs

```diff
@@ -0,0 +1,94 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class SendDataToTheHybridJobTriggerTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public SendDataToTheHybridJobTriggerTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestHybridJob()
+        {
+            var dlp = DlpServiceClient.Create();
+
+            // Create the Job Trigger.
+            var request = new CreateJobTriggerRequest
+            {
+                ParentAsLocationName = new LocationName(_fixture.ProjectId, "global"),
+                JobTrigger = new JobTrigger
+                {
+                    Triggers =
+                    {
+                        new JobTrigger.Types.Trigger
+                        {
+                            Manual = new Manual()
+                        }
+                    },
+                    InspectJob = new InspectJobConfig
+                    {
+                        Actions =
+                        {
+                            new Action
+                            {
+                                JobNotificationEmails = new Action.Types.JobNotificationEmails(),
+                                PublishToStackdriver = new Action.Types.PublishToStackdriver(),
+                            }
+                        },
+                        InspectConfig = new InspectConfig
+                        {
+                            InfoTypes =
+                            {
+                                new InfoType[]
+                                {
+                                    new InfoType { Name = "EMAIL_ADDRESS" }
+                                }
+                            },
+                            MinLikelihood = Likelihood.Possible,
+                            IncludeQuote = true,
+                        },
+                        StorageConfig = new StorageConfig
+                        {
+                            HybridOptions = new HybridOptions
+                            {
+                                Description = "Hybrid job trigger for data from the comments field of a table that contains customer appointment bookings",
+                            },
+
+                        }
+                    }
+                }
+            };
+
+            var hybridJobTrigger = dlp.CreateJobTrigger(request);
+            var text = "My email is ariel@example.org and name is Ariel.";
+            var result = SendDataToTheHybridJobTrigger.SendToHybridJobTrigger(_fixture.ProjectId, hybridJobTrigger.JobTriggerName.JobTriggerId, hybridJobTrigger.Name, text);
+            Assert.Equal(DlpJob.Types.JobState.Active, result.State);
```

Comment:
This might cause flakiness, as maybe the job has completed the first time you query it? Or is that not possible.

Reply:
No, The job created inside the hybrid job will always have state as `Active` and also we are waiting for the job to get completed in sample as well. I don't think this will cause flakiness

Reply:
Well, because you are now using mocks. When I reviewed, you weren't.

And on the sample you are not currently waiting for the job to be completed (that's what I'm proposing in my suggestion), you are waiting for the job to process some amount of bytes.

Reply:
The hybrid job does not have status done; So to check whether the job has completed or we are waiting for the job to process the bytes.